modules/parsers/parser-hdbti/src/main/java/com/sap/xsk/parser/hdbti/models/XSKHDBTIImportConfigModel.java

```diff
@@ -11,90 +11,196 @@
  */
 package com.sap.xsk.parser.hdbti.models;
 
-import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.gson.annotations.SerializedName;
-import lombok.*;
-
 import java.util.ArrayList;
 import java.util.List;
+import java.util.Objects;
 
-@Getter
-@Setter
-@NoArgsConstructor
 public class XSKHDBTIImportConfigModel {
 
-    @SerializedName("table")
-    @JsonProperty("table")
-    private String tableName;
+  @SerializedName("table")
```

Comment:
Why @JsonProperty was removed?

Reply:
@antoanetamh it was replaced by GSON serialization with the `@SerializedName` annotation.

Reply:
Correct, if we are using Gson.
https://www.baeldung.com/json-multiple-fields-single-java-field